src/include/common/error/error_code.h

```diff
@@ -179,6 +179,7 @@ enum class ErrorCode : uint16_t {
   ERRCODE_TOO_MANY_JSON_ARRAY_ELEMENTS,                     // 2203D
   ERRCODE_TOO_MANY_JSON_OBJECT_MEMBERS,                     // 2203E
   ERRCODE_SQL_JSON_SCALAR_REQUIRED,                         // 2203F
+  ERRCODE_DECIMAL_RESULT_OVERFLOW,                          // 22040
```

Comment:
Out of curiosity: where did this come from? We pulled an updated error code list from Postgres when this file was generated a few months ago.

Reply:
I searched overflow and didnt find anything

Reply:
I think the reason is postgres supports arbitrary precision decimals through strings

Reply:
So did we make up that error code?

Reply:
Yes

Reply:
Adding new error codes has larger implications for the system and Postgres compatibility. We should decide as a group what to do if a specific error code we want doesn't exist. Typically you would just defer to the general error code for that class if a more specific one doesn't exist. In this case, it would be `ERRCODE_DATA_EXCEPTION,       // 22000`.

Reply:
was consensus reached?

Reply:
> was consensus reached?

I don't recall it being brought up in a meeting.

Reply:
Resolved in latest PR